include/ck/utility/data_type.hpp

```diff
@@ -1016,4 +1016,43 @@ struct NumericLimits<half_t>
     __host__ __device__ static constexpr half_t Lowest() { return bit_cast<half_t>(binary_lowest); }
 };
 
+template <typename T>
+struct AccumulatorDataType;
```

Comment:
Where is this used for?

If it's used MFMA, it should be implemented either
1.  inside MFMA wrapper, or
2.  by helper function for MFMA

Reply:
It's used here: https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/259/files?show-viewed-files=true&file-filters%5B%5D=#diff-212a6e82200606c612da1b8b47eb2000abcc32a653df6bd74c00d8fda7c27bceL35
I needed this since I actually found a bug, that in the file above I created a template alias for ConvND instance where the `AccDataType` was the same as `InDataType`. Since I didn't wanted to add additional template parameter so I use this trait.

Reply:
```composable_kernel/include``` should only contains the core header file that is needed by clients who use CK as a header only lib.

But this class is only used by CK's internal testing code, so it should be inside ```composable_kernel/include```.

Additionally, we should not assume the ```Acc``` datatype based on input type. This assumption happens to be correct on current GPU, but may or may not be true for future GPUs.

Please remove this class, and explicitly specify Acc data type in the test code